.pylintrc

```diff
@@ -0,0 +1,63 @@
+[MESSAGES CONTROL]
```

Comment:
Instead of having a dot file to control `pylint`, it might be preferable to put its options in `setup.cfg` (and, eventually, `pyproject.toml` if/when we move to PEP621-style packaging).

The `disable` list would go into a `[pylint.messages control]` section of `setup.cfg`.

Reply:
Thanks, that made it prettier. Used `setup.cfg`.